Dockerfile

```diff
@@ -5,10 +5,7 @@ ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 
 WORKDIR /src
 
-COPY vendor vendor
-COPY cmd cmd
-COPY pkg pkg
-COPY Makefile go.mod go.sum ./
+COPY . .
```

Comment:
Is this COPY all a bit excessive?

Reply:
It's only to the intermediary container, our final image size won't change.